plugin/src/main/cpp/export/meta_export_plugin.cpp

```diff
@@ -237,6 +238,22 @@ TypedArray<Dictionary> MetaEditorExportPlugin::_get_export_options(const Ref<Edi
 	return export_options;
 }
 
+Dictionary MetaEditorExportPlugin::_get_export_options_overrides(const Ref<EditorExportPlatform> &platform) const {
+	Dictionary overrides;
+	if (!_supports_platform(platform) || !_is_vendor_plugin_enabled()) {
+		return overrides;
+	}
+
+	HybridType hybrid_type = _get_hybrid_app_option_value();
+	if (hybrid_type != HYBRID_TYPE_DISABLED) {
+		// Unless this is a hybrid app that launches as a panel, then we want to force this option on.
+		// Otherwise, it needs to be off, so that the panel activity can be the one that's launched by default.
+		overrides["package/show_in_app_library"] = (hybrid_type != HYBRID_TYPE_START_AS_PANEL);
+	}
+
+	return overrides;
```

Comment:
Seems this logic should be in the base class since I would expect it to be the same for all supported vendors.

Reply:
So, I tried this, but it's a little awkward when the child class also wants to implement `_get_export_options_overrides()`, because both need to do the same `if (!_supports_platform(platform) || !_is_vendor_plugin_enabled())` logic.

I suppose that's not really a big deal, since none of this code runs very often (only in the export dialog or during an actual export). I can switch back to that.